Playbooks/AS-Slack-Integration/README.md

```diff
@@ -0,0 +1,168 @@
+# AS-Slack-Integration
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Slack-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton"/>
+</a>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Slack-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.png"/>
+</a>
+
+
+This playbook is intended to be run from a Microsoft Sentinel incident. It will create a Slack post with the related Microsoft Sentinel incident information including parsed entity information.
+
+![Slack_Demo](Images/Slack_Demo.png)
+
+
+#
+### Requirements
+
+The following items are required under the template settings during deployment:
+                                                                                                                                     
+* A Slack App Token
+* The Slack Channel Id to post Microsoft Sentinel incident information to
+ 
+For this playbook, you will need to create a Slack Bot app to securely post to your Slack Workspace.                                                                                    
+
+# 
+### Setup
+
+To create a Slack App and obtain a Bot Token:
+                                                                                                                                     
+From your Slack Workspace, navigate to https://api.slack.com/apps/ and click the "**Create a new Slack app**" button.
+
+![Slack_App_1](Images/Slack_App_1.png)
+
+
+Enter in "**Microsoft Sentinel**" for the app name then add a description. 
+Finally, select a workspace and click "**Create App**" .
+
+![Slack_App_2](Images/Slack_App_2.png)
+
+
+Now we need to add permissions to the app in order for it to post messages in channels. 
+From the "**Basic Information**" section of your newly created app, click "**Permissions**".
+
+![Slack_App_3](Images/Slack_App_3.png)
+
+
+Scroll down to the "**Scopes**" section and click the "**Add an OAuth Scope**" button under "**Bot Token Scopes**".
+Select the "**chat:write**" option.
+
+![Slack_App_4](Images/Slack_App_4.png)
+
+
+Scroll back up to the top of the page and click the "**Install to Workspace**" button, which should now be green.
+
+![Slack_App_5](Images/Slack_App_5.png)
+
+
+Click "**Allow**" and then copy the entire value in the "**Bot User OAuth Token**" field. 
+
+![Slack_App_6](Images/Slack_App_6.png)
+
+
+If you would like, you can also add the Microsoft Sentinel Logo under "**Display Information**".

```

Comment:
Indicate where to find the **Display Information**. This option lies under Basic Information section. The user is currently on **OAuth & Permissions**